templates/common/.github/workflows/terraform/azure-dev.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: mcr.microsoft.com/azure-dev-cli-apps:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Log in with Azure
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Azure Dev Configure Environment
+        run: |
+          azd env set RS_RESOURCE_GROUP $RS_RESOURCE_GROUP
```

Comment:
Where does $RS_RESOURCE_GROUP come from in GH?

Reply:
It was referencing the env var, but we don't need it now since its already set on the command.

Reply:
That's manual steps that customers need to follow and to set up `remote-state` for terraform,

Before running `az pipeline config`,  when using terraform templates, users need to do [this](https://review.docs.microsoft.com/en-us/azure/developer/azure-developer-cli/use-terraform-for-azd?branch=pr-en-us-3533).
If the expected env vars are not found in the environment, `az pipeline config` should fail and show a descriptive error for users to first set the env vars, following the setup instructions